composer.json

```diff
@@ -17,8 +16,8 @@
         }
     ],
     "require": {
-        "php": "^7.2|^7.3",
-        "elasticsearch/elasticsearch": ">=7.2",
+        "php": "^7.2|^7.3|^7.4",
+        "elasticsearch/elasticsearch": "^7.2",
         "laravel/scout": "^8.0|^9.0",
```

Comment:
I changed the `>=` with `^` because of `composer validate --strict` validation response. Also, `version` field gave a warning and could not pass from `validate` method of the composer.

Reply:
What type of warning `version` produce?